frontend/src/app/containers/App/index.js

```diff
@@ -63,7 +59,6 @@ import { localizationsSelector, negotiatedLanguagesSelector } from "../../select
 import { chooseTests } from "../../actions/varianttests";
 import addonActions from "../../actions/addon";
 import newsletterFormActions from "../../actions/newsletter-form";
-import RestartPage from "../RestartPage";
```

Comment:
Since the `RestartPage` container is no longer used, it should be removed from the project too